dev-java/swt/swt-4.10-r1.ebuild

```diff
@@ -34,7 +34,7 @@ COMMON_DEP="
 		net-libs/webkit-gtk:4
 	)"
 DEPEND="${COMMON_DEP}
-	>=virtual/jdk-1.8
+	>=virtual/jdk-1.8:*
 	app-arch/unzip
 	virtual/pkgconfig
```

Comment:
these two should probably go to `BDEPEND`

Reply:
the `virtual/jre` should go in `BDEPEND` as well? won't it get dep cleaned?

Reply:
> `irtual/jre` should go in `BDEPE`

not `virtual/jre`. but `app-arch/unzip` and `virtual/pkgconfig` almost always are needed at build time and go to `BDEPEND`.

Reply:
it's as @vaukai wrote. i should have stated that clearly. sorry for that.

Reply:
ahhhh ok, the github UI tricked me here because the diff is highlighted in colors, but I see now that your comment is directly below the line in question. well noted, I will work on fixing this, and keep an eye out for other ebuilds that I bump to EAPI 7

Reply:
fixed